app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<section class="align-center">  
+  <h2>Sign up to continue</h2>
+
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field">
+      <%= f.label :name %><br />
+      <%= f.text_field :name, autofocus: true, autocomplete: "name", class: "input" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "input" %>
+    </div>
```

Comment:
- [ ] I applaud your efforts so far 👏 However, just like the previous reviewers suggested, your form inputs don't look exactly like the given design. It is imperative that you follow the design to the letter. Kindly see images below and make the appropriate adjustments related to  the font-color, font-weight, margins, padding etc
## Design Guideline
![image](https://user-images.githubusercontent.com/46839548/221158125-afce01ff-8367-4ef5-b3bc-366a028fa1b7.png)
## Your Design
![image](https://user-images.githubusercontent.com/46839548/221158547-99cf6d8d-a822-47ba-9aa5-d96e463303e4.png)